.github/workflows/python-client-release-to-pypi.yml

```diff
@@ -0,0 +1,26 @@
+name: Release Python Client to pypi
+
+on:
+  workflow_dispatch: {}
+
+jobs:
+  run-python-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # We technically support 1.16 and 1.18; but this just needs any
+        # go version new enough to generate python proto files; 1.18 is
+        # more future-proof, so use it.
+        go: [ '1.18' ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/workflows/go-setup
+      - uses: ./.github/workflows/python-tests
+        with:
+          python-version: '3.8.10'
+          path: 'client/python'
+      - name: Publish package to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          password: ${{ secrets.PYPI_API_TOKEN }}
+          packages_dir: client/python/dist/
```

Comment:
You should probably parametrize this so we can use this action for airflow-operator also?

Reply:
This is one where copy+change the path will be the easiest way. Only way to manually run a workflow w/inputs is to run them via an API call.